docs/pesin_satis.md

```diff
@@ -13,6 +13,7 @@ $configuration = new Configuration();
 $configuration->setMerchantId('MY_MERCHANT_01')
     ->setSecretKey('SECRET_KEY')
     ->setPaymentEndpointUrl('https://secure.payu.com.tr/order/alu/v3');
+    ->setReturnUrl('http://payu/cs/payu_3dreturn.php');
```

Comment:
Döküman son kullanıcı tarafından değerlendirileceği için buradaki alan adının örnek olduğunu anlamayabilirler. example.com/3d_return.php şeklinde birşeyler olursa sanki biraz daha anlamlı olur gibi duruyor.